### DISCUSSION TITLE
lib/sequel/adapters/postgres.rb:208: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call

### DISCUSSION THREAD

**Discussion by dentarg** [jeremyevans/sequel#1880] [General]
Since pg 1.3.0 (due to https://github.com/ged/ruby-pg/commit/de18b88fb52caee4a40f7bc50b00717cc0ce42b8 I think) the following warning [can be seen on Ruby 2.7](https://github.com/jeremyevans/sequel/runs/6532333705?check_suite_focus=true#step:7:50) (I see it in my app too)

```
/home/runner/work/sequel/sequel/lib/sequel/adapters/postgres.rb:208: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/home/runner/work/sequel/sequel/vendor/bundle/ruby/2.7.0/gems/pg-1.3.5/lib/pg/connection.rb:701: warning: The called method `connect' is defined here
```

Is it something that needs to be addressed?

**Comment by jeremyevans** [comment]
This warning no longer appears in Ruby 3.0, and only shows up when choosing to display deprecation warnings in Ruby 2.7.  As it doesn't show up in Ruby 3.0, the warning must be spurious. Sequel generates many verbose warnings in Ruby before 3.0, and verbose warnings in Sequel when used in Ruby versions below 3 is not considered a bug.

I looked at the ruby-pg code and the problem can be fixed there.  Keywords are never actually used by `connect_start`, so they should drop the use of them in the related methods that delegate to `connect_start`: https://github.com/ged/ruby-pg/pull/456
